### PR TITLE
simplejwt token

### PR DESCRIPTION
**Pull request by visse0001**
WIP

- Added djangorestframework-simplejwt to requirements
- updated README file how we can check if the tokens work :smile: 
- added view `HelloView` that you can't access without a token ( `permission_classes = (IsAuthenticated,)` ) I added a view so we could see how it works. However, there are no views in DRF, so I should remove this before merge.
- added two, new endpoints for token and url for view `HelloView`

- you can also hit http://localhost:8000/api/users/students with token, I added `permission_classes = (IsAuthenticated,)` to StudentsResource class

You may paste token here https://jwt.io/ to see decoded header, payload and verify signature

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,24 @@ Go to http://localhost:8000/admin and create a new user.
...
+### Auth
+
+Go to http://localhost:8000/api/users/token/  
+
+As you can see below, you can fill KEY with username and VALUE with password.  
+After that you should see token, copy that  
+
+Postman
+
+![](https://github.com/zetonteam/zeton_django/blob/auth-with-simplejwt/images/Screenshot_01.png?raw=true)
```
Comment (tgbdc7) [inline README.md:73]: Is super cool, that you paste image. Now it's more understandable.

File: README.md [hunk 2]
```diff
@@ -61,6 +61,24 @@ Go to http://localhost:8000/admin and create a new user.
...
+
+As you can see below, you can fill KEY with username and VALUE with password.  
+After that you should see token, copy that  
+
+Postman
+
+![](https://github.com/zetonteam/zeton_django/blob/auth-with-simplejwt/images/Screenshot_01.png?raw=true)  
+
+Then you can hit http://localhost:8000/api/users/hello  
+with Authorization TYPE Bearer Token - Token - then paste the token
```
Comment (tgbdc7) [inline README.md:76]: Maybe add info which token we mast paste. Refresh or access.

**Review by tgbdc7 — APPROVED** [review]
good job :+1:

File: README.md (hunk 1 above)
Comment (visse0001) [reply README.md:73]: :)

File: README.md (hunk 2 above)
Comment (visse0001) [reply README.md:76]: Okay

**Pull request closed without merging**